gsw_oceanographic_toolbox.c

```diff
@@ -1,6 +1,6 @@
 /*
 **  $Id: gsw_oceanographic_toolbox-head,v c61271a7810d 2016/08/19 20:04:03 fdelahoyde $
-**  Version: 3.05.0-4
+**  Version: 4
```

Comment:
I'm curious as to the meaning of the version.  Moving forward, is there a plan to link C version numbers to the Matlab version numbers, or are these independent?  (I have never truly understood version numbers in GSW, so excuse my ignorance, please.)